command/secrets/init/helper.go

```diff
@@ -0,0 +1,71 @@
+package init
+
+import (
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/network"
+	"github.com/0xPolygon/polygon-edge/secrets"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+// loadValidatorAddress loads ECDSA key by SecretsManager and returns validator address
+func loadValidatorAddress(secretsManager secrets.SecretsManager) (types.Address, error) {
+	if !secretsManager.HasSecret(secrets.ValidatorKey) {
+		return types.ZeroAddress, nil
+	}
+
+	encodedKey, err := secretsManager.GetSecret(secrets.ValidatorKey)
+	if err != nil {
+		return types.ZeroAddress, err
+	}
+
+	privateKey, err := crypto.BytesToECDSAPrivateKey(encodedKey)
+	if err != nil {
+		return types.ZeroAddress, err
+	}
+
+	return crypto.PubKeyToAddress(&privateKey.PublicKey), nil
+}
+
+// loadValidatorAddress loads BLS key by SecretsManager and returns BLS Public Key
+func loadBLSPublicKey(secretsManager secrets.SecretsManager) ([]byte, error) {
+	if !secretsManager.HasSecret(secrets.ValidatorBLSKey) {
+		return nil, nil
+	}
+
+	encodedKey, err := secretsManager.GetSecret(secrets.ValidatorBLSKey)
+	if err != nil {
+		return nil, err
+	}
+
+	secretKey, err := crypto.BytesToBLSSecretKey(encodedKey)
+	if err != nil {
+		return nil, err
+	}
+
+	return crypto.BLSSecretKeyToPubkeyBytes(secretKey)
+}
+
+// loadNodeID loads Libp2p key by SecretsManager and returns Node ID
+func loadNodeID(secretsManager secrets.SecretsManager) (string, error) {
+	if !secretsManager.HasSecret(secrets.NetworkKey) {
+		return "", nil
```

Comment:
No error handling?

Reply:
This function is called only when generating command result. It's ok whether key doesn't exist because this value doesn't show in result.

Reply:
@zivkovicmilos ping